src/HotChocolate/Fusion-vnext/src/Fusion.Composition/Properties/CompositionResources.resx

```diff
@@ -42,6 +42,9 @@
   <data name="LogEntryHelper_ExternalMissingOnBase" xml:space="preserve">
     <value>External field '{0}' in schema '{1}' is not defined (non-external) in any other schema.</value>
   </data>
+  <data name="LogEntryHelper_ExternalOnInterface" xml:space="preserve">
+    <value>Interface '{0}' in schema '{1}' contains a field marked as external.</value>
```

Comment:
How about:

`Interface field 'Node.id' in schema 'A' must not be marked as external.`